src/pages/AboutPage/index.tsx

```
@@ -22,43 +22,41 @@ export default function AboutPage() {
       </div>
-
       <section className="mb-4">
         <h1 className="text-3xl section mb-2">Certificate</h1>
         <div className="grid grid-cols-3 gap-2">
           {certificates.map((cer) => {
             return cer.name !== "invalid" ? (
               <CertificateCard key={cer.name} {...cer} />
             ) : (
-              <div className="flex p-5 border-1 flex-col justify-center items-center"></div>
+              <div className="flex p-5 border-1 flex-col justify-center items-center" />
             );
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)